easybuild/easyconfigs/t/TOML-Fortran/TOML-Fortran-0.4.1-intel-compilers-2022.2.1.eb

```diff
@@ -0,0 +1,25 @@
+easyblock = 'CMakeMake'
+
+name = 'TOML-Fortran'
+version = '0.4.1'
+
+homepage = 'https://github.com/toml-f/toml-f'
+description = 'TOML parser for Fortran projects'
+
+toolchain = {'name': 'intel-compilers', 'version': '2022.2.1'}
+
+source_urls = ['https://github.com/toml-f/toml-f/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['435f6c06dd7392ef282ed7567c9245ca64b8456d62cbf814ada9cbad60f21c00']
+
+builddependencies = [
+    ('binutils', '2.39'),
```

Comment:
binutils is already included in intel-compilers, can be removed?

Reply:
@sassy-crick any thoughts on the comment from @smoors ?